CRITICAL - 2023-10-08 18:10:17 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-10-08 16:53:30 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 16:54:58 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 16:56:30 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 16:57:57 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 17:02:38 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 17:04:48 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-08 17:05:14 --> App\Controllers\Home::index(): Return value must be of type string, none returned
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-08 21:06:12 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'taskappdb'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'taskappdb', Object(SensitiveParameterValue), 'taskapp', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-10-08 21:06:12 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `taskapp`')
 3 SYSTEMPATH\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 APPPATH\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Migrate->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-08 21:52:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'taskappdb'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'taskappdb', Object(SensitiveParameterValue), 'taskapp', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-10-08 21:52:19 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `taskapp`')
 3 SYSTEMPATH\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 APPPATH\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Migrate->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-08 22:20:21 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'taskappdb'@'localhost' (using password: NO) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'taskappdb', Object(SensitiveParameterValue), 'taskapp', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-10-08 22:20:21 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `taskapp`')
 3 SYSTEMPATH\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 APPPATH\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Migrate->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-08 22:20:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'taskapp', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-10-08 22:20:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `taskapp`')
 3 SYSTEMPATH\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 APPPATH\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Migrate->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-08 22:20:52 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'taskappdb'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'taskappdb', Object(SensitiveParameterValue), 'taskapp', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(919): App\Controllers\Migrate->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-10-08 22:20:52 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FROM `taskapp`')
 3 SYSTEMPATH\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
 4 SYSTEMPATH\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
 5 SYSTEMPATH\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
 6 APPPATH\Controllers\Migrate.php(13): CodeIgniter\Database\MigrationRunner->latest()
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Migrate->index()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Migrate))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
